packages/wrangler/src/sites.tsx

```diff
@@ -148,23 +146,55 @@ export async function syncAssets(
     console.log(`reading ${file}...`);
     const content = await readFile(file, "base64");
 
-    const assetKey = await hashAsset(hasher, file, content);
+    const assetKey = hashAsset(hasher, file, content);
     validateAssetKey(assetKey);
 
     // now put each of the files into kv
     if (!keys.has(assetKey)) {
```

Comment:
hmm, this line of code is wrong, it also needs to include assets set to expire 
I'll fix this

Reply:
Ok, fixed.